### ISSUE TITLE
[Quest] Encroaching Darkness III

### ISSUE THREAD

**Issue by JackBinimbul** [WarEmu/WarBugs#5331]
Location: Nordland, Norscan Longships
After completion of the criteria (killing Sea Dogs, finding Pater Allard's body), Encroaching Darkness IV is not triggered.

**Comment by Evildoor** [comment]
Have you tried speaking with NPC who gave you the third part?

**Comment by shibirian** [comment]
Yes, it is bugged.

The final stage IV is not shown on the map! It stays at the beach

**Comment by JackBinimbul** [comment]
No NPC provides part III.  It is triggered via the abandoned sack.  I attempted returning to the NPC who provided the first part of the quest, which allowed completion and obtaining part IV.  This is not indicated on the map or in the quest data as it is intended that Pater's corpse continues the quest. As a result I had to look up whom I even got the original quest from to find him again. Though speaking to the Sigmarite Acolyte does allow continuation as a work around.  Thanks!
